Documentation/ksmbd.adduser.8

```diff
@@ -0,0 +1,35 @@
+.TH KSMBD.ADDUSER "8" "October 2021" "Usage: smbuseradd" "Linux System Administration"
```

Comment:
please change "smbuseradd" to "ksmbd.adduser". This is old name of ksmbd.adduser.

Reply:
i think that's fixed now